app/src/main/java/com/tohami/newsapi/data/repos/NewsRepository.java

```diff
@@ -0,0 +1,63 @@
+package com.tohami.newsapi.data.repos;
```

Comment:
Delete this package, each repository should be included in its component package

Reply:
done